solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -825,6 +896,9 @@ contract StandardPoolConverter is ConverterVersion, IConverter, ContractRegistry
         // set the reserve balances
         setReserveBalances(1, 2, newReserveBalances[0], newReserveBalances[1]);
 
+        // set the reserve balances product
+        __reserveBalancesProd = newReserveBalances[0] * newReserveBalances[1];
```

Comment:
saw this in multiple places, probably worth moving to a utility function

Reply:
It's a 1-liner (which appears in 2 places).